Description: Are non-Windows users not allowed to install this?
Author: mirabilos <tg@debian.org>
Forwarded: no

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,4 +104,6 @@ else (MINGW)
       vorbisenc
       ${SNDFILE_LIB}
    )
+
+   install(TARGETS sfconvert RUNTIME DESTINATION bin)
 endif (MINGW)
